### DISCUSSION TITLE
編集リクエスト不採用後の記事編集で差分が適用されてしまう

### DISCUSSION THREAD

**Discussion by standstonecraft** [increments/qiita-discussions#618] [要望・アイデア・不具合]
編集リクエストをいただき不採用としたあとにその記事を編集しようとすると、編集リクエストの内容が反映された状態になってしまいます。

Chrome拡張が効かない状況で試したり、Edgeで試したりはしましたが、変化ありませんでした。

結果的に編集リクエストを受け入れたので、この記事で起きていると提示することができずすみません。

**Comment by tanaka-asami415** [comment]
@standstonecraft 
ご報告ありがとうございます。
弊社でも試してみたのですが、同様の症状は再現できませんでした🙏

もしかすると、「不採用」ではなく「採用する前に編集する」ボタンを押されてしまったのではないかと考えられるのですが
既に編集リクエストは受け入れられたとのことで、実際どうだったか確認することができないため、もしもまた同症状が出てしまった際は、一度「編集リクエスト管理」の状態を確認してみていただけると幸いです🙏

詳細は以下のヘルプをご確認ください。
https://help.qiita.com/ja/articles/qiita-edit-request#part-70080d39ee900cf2
